pkg/image/importer/importer.go

```diff
@@ -217,11 +225,20 @@ func (i *ImageStreamImporter) importFromRepository(ctx gocontext.Context, retrie
 	if from.Kind != "DockerImage" {
 		return
 	}
-	ref, err := api.ParseDockerImageReference(from.Name)
-	if err != nil {
-		status.Status = invalidStatus("", field.Invalid(field.NewPath("from", "name"), from.Name, fmt.Sprintf("invalid name: %v", err)))
-		return
+	var (
```

Comment:
@smarterclayton OK, I will. But our client still [sends](https://github.com/openshift/origin/blob/master/pkg/generate/app/dockerimagelookup.go#L197) this symbol to see the list.
why this workaround should be removed ?